simde/arm/neon/ld1.h

```diff
@@ -430,6 +431,122 @@ simde_vld1q_u64(uint64_t const ptr[HEDLEY_ARRAY_PARAM(2)]) {
   #define vld1q_u64(a) simde_vld1q_u64((a))
 #endif
 
+SIMDE_FUNCTION_ATTRIBUTES
+simde_poly8x8_t
+simde_vld1_p8(simde_poly8_t const ptr[HEDLEY_ARRAY_PARAM(8)]) {
```

Comment:
No test?

Reply:
No independent test for the `ld1` function (Since `simde/test/arm/neon/meson.build` excluded `ld1`, `ld3`, `ld4`, and `st1`), but this function will be called by another function test. For example, lines 917 in file `ld1_x3.c`.